[2022-02-15T17:04:52.294079] Using urllib.request Python 3.0 or later
Streaming log file azureml-logs/60_control_log.txt
Running: ['cmd.exe', '/c', 'C:\\Users\\ag4488\\AppData\\Local\\Temp\\azureml_runs\\experiment-pysdk-by-script_1644944689_ae4385ae\\azureml-environment-setup/conda_env_checker.bat']
Starting the daemon thread to refresh tokens in background for process with pid = 28332
Found materialized conda environment in: C:\Users\ag4488/.azureml/envs/azureml_921df1ff867988d12a208a6e95639795


Running: ['C:\\Users\\ag4488/.azureml/envs/azureml_921df1ff867988d12a208a6e95639795/python.exe', 'azureml-setup/run_script.py', '$USERPROFILE/.azureml/envs/azureml_921df1ff867988d12a208a6e95639795/python.exe', 'azureml-setup/context_manager_injector.py', '-i', 'ProjectPythonPath:context_managers.ProjectPythonPath', '-i', 'RunHistory:context_managers.RunHistory', '-i', 'TrackUserError:context_managers.TrackUserError', 'exp_sample_dataset.py']
[2022-02-15T17:04:52.542044] Logging experiment running status in history service.
Streaming log file azureml-logs/70_driver_log.txt
Starting the daemon thread to refresh tokens in background for process with pid = 292
Script process exited with code 0
Uploading driver log...
Finalizing run...
[2022-02-15T12:05:37.848295] get vm size and vm region successfully.
[2022-02-15T12:05:37.859296] get compute meta data successfully.
[2022-02-15T12:05:38.307403] post artifact meta request successfully.
[2022-02-15T12:05:38.549419] upload compute record artifact successfully.

Script process exited with code 0



Uploading control log...
